docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -46,106 +76,81 @@ At a high level, you will want to track these 3 categories of metrics:
-We recommend doing this for all your Workflows and Activities in production.
-
-You can go even farther by defining your inputs and outputs using [Protocol Buffers (protobuf)](https://protobuf.dev/) or another interface definition language (IDL).
-We support protobuf with built-in converters.
-
-### Dealing with very long-lived or very event-heavy Workflows
-
-If you know that your Workflow will live for a very long time or generates a large volume of [Events](/workflows#event), you need to deal with the Event [limits](/workflows#limits) and [Continue-As-New](/workflows#continue-as-new) at some point.
-
-One technique to prepare for this eventuality (and at the same time make your life easier when you need to make changes to the [Workflow Definition](/workflows#workflow-definition)) is to structure your Workflow as an "entity Workflow."
-
-### Make all non-transient state serializable
-
-You already defined your Workflow input as an object or struct like we mentioned in the previous section, right?
-If you didn't, do that.
-
-The struct should be capable of representing all state that your Workflow cares about.
-Because it is also your Workflow input, it must be serializable.
+## Availability
 
-In practice, this often ends up looking like a state machine.
+A high level of availability and reliability (99.99%) is a requirement for mission critical deployments.
+We recommend testing for this availability level while load testing.
+We also recommend validating this level of reliability while doing server upgrades, to ensure no loss of service availability.  
 
-### Continue-As-New on demand
+Temporal Clusters can be deployed in as many regions as needed to meet various requirements:
+- Data Residency
+- Latency
+- Security / Isolation
+This can multiply the effort to implement and maintain clusters.
 
-Set up a mechanism to force your Workflow to Continue-As-New on demand.
-The easiest way to do this is to define a Signal handler that continues the Workflow when received.
-Be sure that all previously received Signals have been processed and that all running [Child Workflows](/encyclopedia/child-workflows) have been waited on or otherwise dealt with.
+[Temporal Cloud is available in 15 AWS regions](/cloud/service-availability).
 
-The newly started Workflow then unpacks the state in the input and can "resume" the Workflow according to your business logic.
+## Management and Control Plane 
 
-A huge benefit of this approach is that you avoid making patches to running Workflows.
-Instead, tell them all to continue onto a new [Task Queue](/workers#task-queue) with [Workers](/workers) that have the updated Workflow code.
+When you experience success with Temporal, you will typically expand your deployment.
```

Comment:
Throwing this in, although it's so "marketing": "Temporal success leads to larger Temporal deployments. When you experience how well Temporal solves your problems and lowers costs, the natural response is..." Thought it might be inspirational as a comment.